repokitteh.star

```diff
@@ -24,6 +24,13 @@ use(
       "github_status_label": "any API change",
       "auto_assign": True,
     },
+    {
+      "owner": "envoyproxy/mobile-maintainers!",
```

Comment:
```suggestion
      "owner": "envoyproxy/mobile-maintainers",
```

not sure if this is correct 8/

Reply:
dependency and api shephards have the ! so I assume it's useful?  API watchers does not but it doesn't auto-assign
cc @itayd

Reply:
iirc, and from looking at the code (https://github.com/envoyproxy/envoy/blob/main/ci/repokitteh/modules/ownerscheck.star),  '!' means this requires approval and not only an FYI. @htuch actually rewrote the entire owners thing so he might know better.

Reply:
I think that's "allow global approval" because dependency-shepherds has a ! and doesn't  have the approval problem.  Anyway if no one knows and @phlax prefers no ! I'll remove.  We can always fix up later if it's a problem.

Reply:
im happy either way - just wasnt sure whether it was correct - reading the code here https://github.com/envoyproxy/envoy/blob/d1be114907c9513b1b45a5eef5b0a88b6188ec7c/ci/repokitteh/modules/ownerscheck.star#L148-L155 its not immediately obvious what it is doing - i can dig through the code and test to clarify

Reply:
follow up here #24241